### PR TITLE
Unit tests

### PR DESCRIPTION
**Pull request by kleesc**

### CODE REVIEW

File: image/image.go [hunk 1]
```diff
@@ -210,6 +214,11 @@ func ParseContainerStatus(containerStatus v1.ContainerStatus) (*Image, error) {
 	// Set container id
 	image.ContainerID = containerStatus.ContainerID
 
+	// Check if image was pulled by digest or tag
+	if len(strings.SplitN(containerStatus.Image, "@", 2)) > 1 {
+		return image, nil
+	}
```
Comment (kleesc) [inline image/image.go]: Don't set the tag if it was pulled by digest.

File: labeller/labeller.go [hunk 2]
```diff
@@ -102,10 +103,15 @@ func New(config *Config, kubeconfig string, logger log.Logger) (*Labeller, error
 	l.queue = workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), "labeller")
 
 	multiNamespacePodListWatcher := NewMultiNamespaceListerWatcher(
-		kclient.CoreV1().RESTClient(),
-		"pods",
 		l.namespaces,
-		fields.Everything(),
+		func(namespace string) cache.ListerWatcher {
```
Comment (alecmerdler) [inline labeller/labeller.go]: We're making `ListerWatchers` directly now?

File: labeller/manifest.go [hunk 3]
```diff
@@ -196,6 +196,7 @@ func addAffectedPod(key, containerID string, manifest *secscanv1alpha1.ImageMani
 	if containerIds, ok := manifest.Status.AffectedPods[key]; ok {
 		if !contains(containerIds, containerID) {
 			containerIds = append(containerIds, containerID)
+			manifest.Status.AffectedPods[key] = containerIds
```
Comment (alecmerdler) [inline labeller/manifest.go]: Nice catch

File: labeller/labeller.go (hunk 2 above)
Comment (kleesc) [reply labeller/labeller.go]: It's so we can pass the fakes' list/latch function directly for testing. I don't think the `RESTClient` that was being passed to `NewListerWatcherFromClient` to generate a `ListerWatcher`s was being initialized correctly.

**Pull request merged**